app/controllers/concerns/enforce_question_order.rb

```diff
@@ -95,6 +96,14 @@ def ask_for_ni_number?
     trn_request.has_ni_number && trn_request.ni_number.nil?
   end
 
+  def ask_for_trn?
+    return false unless FeatureFlag.active?(:ask_for_trn)
```

Comment:
We also shouldn't ask the user for their TRN if they are not in the Identity flow; see above comment.